api/client/trust.go

```diff
@@ -156,7 +156,7 @@ func (cli *DockerCli) getNotaryRepository(repoInfo *registry.RepositoryInfo, aut
 	}
 
 	creds := simpleCredentialStore{auth: authConfig}
-	tokenHandler := auth.NewTokenHandler(authTransport, creds, repoInfo.CanonicalName, "push", "pull")
+	tokenHandler := auth.NewTokenHandler(authTransport, creds, repoInfo.CanonicalName, []string{"push", "pull"})
```

Comment:
p.s. Don't make `getNotaryRepository` take a `...string`, it's still early days for trust and that would potentially make changes harder.